src/python-common/ceph/deployment/drive_selection/selector.py

```diff
@@ -1,7 +1,9 @@
 import logging
 
+from orchestrator import DaemonDescription
```

Comment:
```
ceph/deployment/drive_selection/selector.py:3: in <module>
    from orchestrator import DaemonDescription
E   ModuleNotFoundError: No module named 'orchestrator'
```

resolves locally for me.. How to resolve that properly?

Reply:
you should not import MGRs modules from python-common. note that python-common code will be imported by ceph-volume

Reply:
done